build/codegen/rook-client-python/README.md

```diff
@@ -0,0 +1,67 @@
+# rook-client-python
+
+Automatically generated models for Rook's custom resource definitions.
+
+Right now, it supports three operators:
```

Comment:
What is missing to support other operators? The crd validation?

Reply:
Hm. `build/codegen/rook-client-python` has no dependency to Go etc. If you're a pure python developer, you don't need to call make to develop with the Pyhton API.